docs/proposals/docker-event-collection.md

```diff
@@ -0,0 +1,86 @@
+# Docker Event Collection
+## Overview
+Docker events are not collected or stored by the Openshift cluster.  Certain customers are [required](https://trello.com/c/qxoFsKz9/) to audit these events (e.g. security).  This document is the architecture and design proposal for gathering Docker events for persistence.
+
+## Requirements
+### Security 
+Docker events shall only be visible by users in a cluster admin role.
```

Comment:
s/Docker events/Docker event log entries/, and s/cluster admin role/just as other operational log entries are visible/.